### PR TITLE
creating the Options subdialog

### PR DESCRIPTION
**Pull request by nyagishira**

### CODE REVIEW

File: instat/sdgOptions.vb [hunk 1]
```diff
@@ -1,3 +1,37 @@
 ﻿Public Class sdgOptions
+    Private Sub sdgOptions_Load(sender As Object, e As EventArgs) Handles MyBase.Load
 
+    End Sub
+
+    Private Sub tbpDisplay_Click(sender As Object, e As EventArgs) Handles tbpDisplay.Click
+
+    End Sub
+
+    Private Sub CheckBox2_CheckedChanged(sender As Object, e As EventArgs) Handles chkANOVATable.CheckedChanged
```
Comment (dannyparsons) [inline instat/sdgOptions.vb]: What are all these check boxes for? Are they needed?

Comment (nyagishira) [inline instat/sdgOptions.vb]: i want to create one subdialog that will be be called in alot of the ANOVA
and Regression dialogs then create methods to hide or show various
controlls depending on the dialog

**Comment by dannyparsons** [comment]
What dialog is the subdialog for? It should have a more specific name as we will many subdialogs.

**Comment by nyagishira** [comment]
call me on skype please. my id is nyambura_mutahi

**Comment by nyagishira** [comment]
check the changes i have made

**Pull request merged**